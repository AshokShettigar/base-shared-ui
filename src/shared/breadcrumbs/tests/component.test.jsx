import { shallow } from 'enzyme';
import PageBreadcrumb from '../component';

const defaultProps = {
    breadcrumbs: [{
        title: 'previous breadcrumb title',
        link: '/link'
    },
    {
        title: 'current breadcrumb title'
    }]
};

describe('Page breadcrumb', () => {
    it('renders with default props', () => {
        const wrapper = shallow(<PageBreadcrumb {...defaultProps} />);
        expect(wrapper).toMatchSnapshot();
    });
});
