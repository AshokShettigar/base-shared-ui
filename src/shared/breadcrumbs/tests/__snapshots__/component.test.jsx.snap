// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page breadcrumb renders with default props 1`] = `
<div
  className="root"
  data-component="Breadcrumbs"
>
  <Link
    className="backLink"
    replace={false}
    to="/link"
  >
    <Icon
      className="backIcon"
      icon="caret-left"
      size="extra-small"
    />
  </Link>
  <div
    className="pageHeader"
  >
    <span
      className="breadcrumbs"
    >
      <Link
        className="breadcrumb"
        replace={false}
        to="/link"
      >
        PREVIOUS BREADCRUMB TITLE
      </Link>
      <span
        className="currentBreadcrumb"
      >
         / CURRENT BREADCRUMB TITLE
      </span>
    </span>
    <div
      className="title"
    >
      current breadcrumb title
    </div>
  </div>
</div>
`;
